app/3.0/docs/polyfills.md

```diff
@@ -4,18 +4,14 @@ title: Polyfills
 
 <!--toc -->
 
-<div>
-{% include 'outdated.html' %}
-</div>
-
-Polymer 2.x has been developed alongside and tested with a new suite of v1-spec compatible polyfills
-for custom elements and shadow DOM. You can test Polymer 2.x by using the latest 1.0 version of
-`webcomponentsjs`, which is included as a bower dependency to Polymer 2.x. (`webcomponentsjs`
+Polymer 3.x has been developed alongside and tested with a new suite of v1-spec compatible polyfills
+for custom elements and shadow DOM. You can test Polymer 3.x using the latest 2.0 version of
```

Comment:
should we mention explicitly that the HTML Imports polyfill has been removed from v2.0 & there is no webcomponents-lite.js?

Reply:
& maybe even that you need to do this in polymer.json if using loader?

```
"extraDependencies": [
    "node_modules/@webcomponents/webcomponentsjs/bundles/**"
  ]
```